buddyboss/buddyboss-platform
# Fixed Some UI issues on Album page

### All Submissions:


Fixes #513 .

### How to test the changes in this Pull Request:

1. Go to User Album
2. Click on Select Photos from Library
3. Please check, all issues are fixed.

### Proof Screenshots or Video

[Done]
https://prnt.sc/r5k9wz
https://prnt.sc/r5l235
https://prnt.sc/r5lyj6

Find a new Issue in smaller device like iphone-5 & 6/7/8
**Before** > https://prnt.sc/r5kyea
**After** > https://prnt.sc/r5kzdo


### Other information:
